#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kernel_size", int_t, 0, "kernel_size", 11, 1, 100)
gen.add("sigma_x", double_t, 0, "sigma_x", 10, 1, 100)
gen.add("sigma_y", double_t, 0, "sigma_y", 10, 1, 100);

exit(gen.generate(PACKAGE, "jsk_perception", "GaussianBlur"))
